sdk/ios/Configsum/Configsum/Configsum/Persistance/TransientUserDefaults.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  TransientUserDefaults.swift
+//  Configsum
+//
+//  Created by Alexandru Savu on 2017-10-05.
+//  Copyright © 2017 Lifesum. All rights reserved.
+//
+
+import Foundation
+
+fileprivate class TransientUserDefaults: UserDefaults {
```

Comment:
do we need to keep it in the Main Target? is it only used in Tests?

Reply:
this is used in `Environment` therefore it's target must be Configsum target